link.html

```diff
@@ -253,25 +253,97 @@ <h1>nbgitpuller link generator<a class="headerlink" href="#nbgitpuller-link-gene
           </div>
         </div>
 
-        <div class="form-group row" id="content-repo-group" hidden="true">
+        <div class="form-group row" id="app-content-provider">
+          <div class="col-sm-2 col-form-label">
+            <label for="app-content-provider" class=>Notebook Content Provider</label>
+            <small class="form-text text-muted">
+            </small>
```

Comment:
```suggestion
            <label>Notebook Content Provider</label>
```

Reply:
are you wanting to get rid of the for="xxx" here? The "class=" is gone yes.